ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -92,7 +92,10 @@
         provider-id (or (:provider params)
                         (:provider-id route-params))
         native-id (:native-id route-params)
-        ; TODO: Generic work - add token check
+        _ (lt-validation/validate-launchpad-token request-context)
+        _ (api-core/verify-provider-exists request-context provider-id)
+        _ (acl/verify-ingest-management-permission
+         request-context :update :provider-object provider-id)
```

Comment:
is this how the editor spaced things?

Reply:
we taled and decided to leave this for now.